website/addons/evernote/__init__.py

```diff
@@ -0,0 +1,49 @@
+from website.addons.evernote import model, routes
+
+# MUST
+
+SHORT_NAME = 'evernote'
+FULL_NAME = 'Evernote'
+
+ROUTES = [routes.api_routes]
+
+MODELS = []
+
+ADDED_DEFAULT = []
+ADDED_MANDATORY = []
+
+# make the evernote addon show up as a widget like Zotero, Mendeley
+VIEWS = ['widget']
+
+# does this setting make it show up in the Files section?
```

Comment:
Yes. `HAS_HGRID_FILES` will need to be set to `True` after adding appropriate providers to WB/MFR